lib/gateway/performance_platform_gateway.rb

```diff
@@ -0,0 +1,20 @@
+class PerformancePlatformGateway
+  def send_stats(data)
+    uri = URI("#{ENV['PERFORMANCE_URL']}data/gov-wifi/volumetrics")
+
+    post(uri, data)
+  end
+
+private
+
+  def post(uri, data)
+    request = Net::HTTP::Post.new(uri)
+    request['Authorization'] = "Bearer #{ENV['PERFORMANCE_BEARER_VOLUMETRICS']}"
```

Comment:
This gateway is currently coupled to volumetrics.

Reply:
Yeah, I don't want to go too far with over-architecting this now. When we add different stats, only then we will think about how to DI them to the gateway.